include/wx/unix/glegl.h

```diff
@@ -125,6 +125,7 @@ class WXDLLIMPEXP_GL wxGLCanvasEGL : public wxGLCanvasBase
     // Only called when using Wayland to indicate that we should be redrawn.
     void OnWLFrameCallback();
 
+    bool m_wlPendingMap;
```

Comment:
Don't we need to initialize it to `false` here?

Reply:
If my understanding is correct, it's not necessary because you're guaranteed to get a `"map"` signal (which initializes the flag) before a `"map-event"` (which reads the flag). Though it's probably not a bad idea to initialize it anyway, just in case...